src/pe_reports/report_gen/views.py

```diff
@@ -0,0 +1,181 @@
+    report_date = False
+    output_directory = False
+
+    formExternal = InfoFormExternal()
+
+    if formExternal.validate_on_submit() and formExternal.submit.data:
+        LOGGER.info("Got to the submit validate")
+        report_date = formExternal.report_date.data
+        output_directory = formExternal.output_directory.data
+        formExternal.report_date.data = ""
+        formExternal.output_directory.data = ""
+
+        if not validate_date(report_date):
+            flash(
+                "Incorrect data format, should be YYYY-MM-DD or not correct report date",
+                "warning",
+            )
+            return redirect(url_for("report_gen.report_gen  "))
+
+        if not os.path.exists(output_directory):
+            os.mkdir(output_directory)
+
+        # Generate reports
+        generate_reports(report_date, output_directory)
+
+    bulletinForm = BulletinFormExternal()
+
+    if bulletinForm.validate_on_submit() and bulletinForm.submit1.data:
+        LOGGER.info("Submitted Bulletin Form")
+
+        id = bulletinForm.cybersix_id.data
+        user_input = bulletinForm.user_input.data
+        output_dir = bulletinForm.output_directory1.data
+        file_name = bulletinForm.file_name.data
+        bulletinForm.cybersix_id.data = ""
+        bulletinForm.user_input.data = ""
+        bulletinForm.output_directory1.data = ""
+        bulletinForm.file_name.data = ""
+
+        file_name = file_name.replace(" ", "")
+        if not validate_filename(file_name):
+            flash(
+                "Invalid filename entered, please enter a different filename",
+                "warning",
+            )
+            return redirect(url_for("report_gen.report_gen"))
+
+        if not os.path.exists(output_dir):
+            flash(
+                "Invalid output directory provided, please enter an existing directory",
+                "warning",
+            )
+            return redirect(url_for("report_gen.report_gen"))
+
+        generate_cybersix_bulletin(id, user_input, output_dir, file_name)
+
+    credsForm = CredsFormExternal()
+    if credsForm.validate_on_submit() and credsForm.submit2.data:
+        breach_name = credsForm.breach_name.data
+        org_id = credsForm.org_id.data
+        credsForm.breach_name.data = ""
+        credsForm.org_id.data = ""
+        all_orgs = get_orgs_df()
+        all_orgs = all_orgs[all_orgs["report_on"] == True]
+
+        if org_id != "":
+            org_id = org_id.upper()
+            all_orgs = all_orgs[all_orgs["cyhy_db_name"].str.upper() == org_id]
+
+        if len(all_orgs) < 1:
+            flash(
+                "The provided org_id does not exist in the database, try another.",
+                "warning",
+            )
+            return redirect(url_for("report_gen.report_gen"))
+
+        for org_index, org in all_orgs.iterrows():
+            LOGGER.info(f"Running on {org['name']}")
```

Comment:
Remove f-string from log message:
```suggestion
            LOGGER.info("Running on %s", org['name'])
```